download.py

```diff
@@ -0,0 +1,53 @@
+import os
+import wget
+import requests
+from github import Github
```

Comment:
We can remove this line and the `pyGithub` repo from requirements?

Reply:
yes, well seen